ld/STM32F429ZI.ld

```diff
@@ -245,4 +245,10 @@ SECTIONS
     PROVIDE(__heap_size = SIZEOF(.heap));
     PROVIDE(__mbed_sbrk_start = ADDR(.heap));
     PROVIDE(__mbed_krbs_start = ADDR(.heap) + SIZEOF(.heap));
+
+    /* Provide physical memory boundaries for uVisor. */
+    __uvisor_flash_start = ORIGIN(VECTORS);
+    __uvisor_flash_end = ORIGIN(FLASH) + LENGTH(FLASH);
+    __uvisor_sram_start = ORIGIN(CCM);
```

Comment:
Am I missing something ? For k64f , you reduced the start by RAM vectors, which should be the same case for this target? Can't locate the PR, was merged yesterday, but I recall that number 0x200 there.